src/components/Header.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect } from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import DecemberBankIcon from '@mui/icons-material/Paid';
+import { useAppDispatch, useAppSelector } from '../redux/hooks';
+import { getUserInfo } from '../api/account/AccountApi';
+import { loadUserInfo, unLoadUserinfo } from '../redux/slices/accountSlice';
+import { endSession } from '../redux/slices/sessionSlice';
+
+export default function Header() {
+  const dispatch = useAppDispatch();
+  const user = useAppSelector((state) => state.account.user);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const us = await getUserInfo();
+        dispatch(loadUserInfo(us));
+      } catch (e: any) {
+        dispatch(unLoadUserinfo());
+      }
+    })();
+  }, [user]);
+
+  const clickLogoutHandler = (e: any) => {
+    e.preventDefault(e);
```

Comment:
Si mal no me acuerdo, el `preventDefault` no requiere argumento
```suggestion
    e.preventDefault();
```